ui/libproject/devstart.rb

```diff
@@ -89,6 +96,12 @@ def clean_git_hooks()
 })
 
 Common.register_command({
+  :invocation => "rebuild-image",
```

Comment:
That's fair. I'm going to merge this and follow-up with something, since I can't think of exactly what I want that to look like yet.